technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note.csl

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Technische Universität Dresden - Finanzwirtschaft und Finanzdienstleistungen (note)</title>
+    <title-short>TUD Betriebswirtschaftslehre (note)</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard1" rel="template"/>
+    <link href="https://bildungsportal.sachsen.de/opal/url/RepositoryEntry/683835403" rel="documentation"/>
+    <author>
+      <name>Jens Mittelbach</name>
+      <email>jens.mittelbach@slub-dresden.de</email>
+      <uri>http://www.slub-dresden.de</uri>
+    </author>
+    <category citation-format="note"/>
+    <category field="social_science"/>
+    <summary>Zitierstil entsprechend den Zitierrichtlinien des Lehrstuhls Finanzwirtschaft und Finanzdienstleistungen der Fakultät Wirtschaftswissenschaften, Technische Universität Dresden.</summary>
+    <updated>2015-11-14T10:58:54+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="available at">Internet</term>
+      <term name="accessed">Stand</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" font-variant="normal" delimiter=", ">
+      <name font-variant="small-caps" delimiter="; " et-al-min="0" initialize="false" name-as-sort-order="all"/>
+      <label form="short" font-variant="normal" prefix=" (" suffix="):"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author" font-variant="normal">
+      <name font-variant="small-caps" delimiter="; " delimiter-precedes-last="never" et-al-min="0" initialize="false" initialize-with="." name-as-sort-order="all"/>
```

Comment:
if you could remove the `et-al-min="0"` here, too? (and above in line 29). This applies to all your current style submissions. From my side that's the last issue, everything else looks good.